### ISSUE TITLE
Error on "Ask a question"

### ISSUE THREAD

**Issue by dantodor** [arsenikstiger/dremio-driver#2]
First of all, thanks for the great work!
I connected Metabase to Dremio, and it worked without a hitch. 
Still, when trying to browse, I'm getting this error:
```
VALIDATION ERROR: Table \'mytest\' not found
SQL Query -- Metabase:: userID: 1 queryType: MBQL queryHash:
 9ca912290e54faa0306ac23bb83857a9a5c04317722c9709cc15014b01823f75
SELECT \"mytest\".\"messages\".\"connectionId\" AS \"connectionId\", .....
```
The thing is, `mytest` is the Dremio datasource, and `messages` is the table. So, I think the error comes from how the query is constructed, it should be without the data source name.
NB, while using manual queries, works just great.
Is there a way by which I can avoid prefixing with datasource name ?

**Comment by dfdf** [comment]
Hi @dantodor wich version o metabase are you using? I know this is not your answer but I'm trying everything to use this connector and it just not work.

**Comment by dantodor** [comment]
@dfdf Latest I got from their website, installed it 2 or 3 weeks ago. Now, the thing is, in order to make it work, you need to use Java 8, so I had to downgrade the Java version. It's not from Metabase, it's from Dremio. For whatever reason, their driver, including the one that's used with this repository, won't work unless it's in a Java 8 JVM. Pretty weird, if you ask me, but it is what it is. I would also like to understand why they have chosen not to publish the driver to a Maven repository, but make you download it. So you can't just add it to a project. Great project, great engine, bad small decisions that are close to make it unusable. They probably want you to pay, but still keep the aura of open source.

**Comment by dfdf** [comment]
I totally agree with you, Dremio is an awesome tool but they are heavy focused only in PowerBI (due to they recent partnership with microsoft) and Tableau, both expensive BI Tools. Thank you a lot for your solution, you really helped me, I was totally lost trying to make this works. I gonna do some test soon and come back here if I find the solution for your problem.
